src/components/tables/cells/stats/Bytes.tsx

```diff
@@ -1,23 +1,62 @@
-import { TableCell } from '@mui/material';
-import { tableBorderSx } from 'context/Theme';
+import { Box, TableCell, Tooltip, Typography } from '@mui/material';
+import {
+    semiTransparentBackgroundIntensified,
+    tableBorderSx,
+} from 'context/Theme';
 import prettyBytes from 'pretty-bytes';
+import { useMemo } from 'react';
+import { useIntl } from 'react-intl';
 
 interface Props {
-    val?: number;
+    read?: boolean;
+    val?: number | null;
 }
 
-const Bytes = ({ val }: Props) => {
+// TODO (stats) we can combine this with the Docs component.
+const Bytes = ({ read, val }: Props) => {
+    const intl = useIntl();
+    const statsLoading = val === null;
+    const defaultedVal = val ?? 0;
+    const number = useMemo(
+        () =>
+            prettyBytes(defaultedVal, {
+                minimumFractionDigits: 2,
+                maximumFractionDigits: 2,
+            }),
+        [defaultedVal]
+    );
+
     return (
         <TableCell
             sx={{
                 ...tableBorderSx,
+                minWidth: 'min-content',
                 maxWidth: 'min-content',
             }}
         >
-            {prettyBytes(val ?? 0, {
-                minimumFractionDigits: 2,
-                maximumFractionDigits: 2,
-            })}
+            <Box sx={{ maxWidth: 'fit-content' }}>
+                <Tooltip
+                    title={`${defaultedVal} ${intl.formatMessage({
+                        id: read
+                            ? 'entityTable.stats.bytes_read'
+                            : 'entityTable.stats.bytes_written',
+                    })}`}
+                >
+                    <Typography
+                        sx={{
+                            transitionDelay: statsLoading ? '800ms' : '0ms',
```

Comment:
Used to prevent the entire table from flashing quickly when a user with fast network changes the range.